tools-local/tasks/core-setup.tasks.csproj

```diff
@@ -34,6 +34,9 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' != 'net45'">
     <Compile Include="netstandard/FileUtilities.netstandard.cs" />
+
+    <Reference Include="$(BuildToolsTaskCoreDir)Microsoft.DotNet.Build.CloudTestTasks.dll" />
```

Comment:
I think in this case he needs to use the independent properties so he can control if he gets the netfx or netstandard version of the task library.